### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by aahung**

### CODE REVIEW

File: Unshaky/ja.lproj/Preference.strings [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+"JiA-hI-REz.headerCell.title" = "ディレイ (ms), 推奨値40";
+
+/* Class = "NSWindow"; title = "Unshaky Preference"; ObjectID = "KI8-Ox-xeb"; */
+"KI8-Ox-xeb.title" = "Unshaky 環境設定";
+
+/* Class = "NSTextFieldCell"; title = "-"; ObjectID = "NxQ-OB-lir"; */
+"NxQ-OB-lir.title" = "-";
+
+/* Class = "NSTextFieldCell"; title = "Set ALL to (ms)"; ObjectID = "Qnt-tS-QWb"; */
+"Qnt-tS-QWb.title" = "全てを(ms)に設定します";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Preference.strings]:
```suggestion
"Qnt-tS-QWb.title" = "全てのディレイを設定します (ms)";
```

File: Unshaky/ja.lproj/Preference.strings [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+/* Class = "NSTextFieldCell"; title = "-"; ObjectID = "Vl3-75-cL7"; */
+"Vl3-75-cL7.title" = "-";
+
+/* Class = "NSButtonCell"; title = "Aggressive mode (experimental)"; ObjectID = "Zqq-UB-oIK"; */
+"Zqq-UB-oIK.title" = "アグレッシブモード (试行)";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Preference.strings]:
```suggestion
"Zqq-UB-oIK.title" = "アグレッシブモード (実験)";
```

File: Unshaky/ja.lproj/Preference.strings [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+"BTT-FQ-Yii.title" = "コピー";
+
+/* Class = "NSButtonCell"; title = "Cancel"; ObjectID = "CUy-rX-LgD"; */
+"CUy-rX-LgD.title" = "キャンセル";
+
+/* Class = "NSButtonCell"; title = "Export"; ObjectID = "Dr8-2D-to4"; */
+"Dr8-2D-to4.title" = "エクスポート";
+
+/* Class = "NSButtonCell"; title = "Fix if CMD+SPACE not work (experimental)"; ObjectID = "I7x-fY-0vx"; */
+"I7x-fY-0vx.title" = "CMD+SPACEの不具合をフィックスします (试行)";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Preference.strings]:
```suggestion
"I7x-fY-0vx.title" = "CMD+SPACEが機能しない不具合を修正します (実験)";
```
We never say 不具合をフィクスします,  フィクス(fix) is a loanword but it usually have  different meanings.
> [Sometimes a loanword develops a different meaning or usage than its donor word.](https://www.thelanguageisland.com/common-japanese-loanwords/)

File: Unshaky/ja.lproj/Preference.strings [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+"Zqq-UB-oIK.title" = "アグレッシブモード (试行)";
+
+/* Class = "NSButtonCell"; title = "Paste"; ObjectID = "acO-QS-DTH"; */
+"acO-QS-DTH.title" = "ペースト";
+
+/* Class = "NSTableColumn"; headerCell.title = "Key"; ObjectID = "asf-bP-xrf"; */
+"asf-bP-xrf.headerCell.title" = "キー";
+
+/* Class = "NSButtonCell"; title = "Ignore external keyboard (experimental)"; ObjectID = "dE9-oM-Tqa"; */
+"dE9-oM-Tqa.title" = "外付けキーボードを無視します (试行)";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Preference.strings]:
```suggestion
"dE9-oM-Tqa.title" = "外付けのキーボードを無視します (実験)";
```

File: Unshaky/ja.lproj/MainMenu.strings [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+*/
+
+/* Class = "NSMenuItem"; title = "Open debug window"; ObjectID = "Iv3-MX-v73"; */
+"Iv3-MX-v73.title" = "デバッグ画面を開きます";
+
+/* Class = "NSMenuItem"; title = "-"; ObjectID = "T9Q-qI-cKF"; */
+"T9Q-qI-cKF.title" = "-";
+
+/* Class = "NSMenuItem"; title = "Configure"; ObjectID = "TGg-Zy-iYa"; */
+"TGg-Zy-iYa.title" = "コンフィグ";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/MainMenu.strings]:
```suggestion
"TGg-Zy-iYa.title" = "設定";
```

File: Unshaky/ja.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+  Created by forgazer on 2/26/19.
+  Copyright © 2019 Nested Error. All rights reserved.
+*/
+
+"Version" = "バージョン: %@";
+
+"Overall Statistic" = "二重入力の取り消し %d 回";
+
+"Configuration Window Title" = "Unshaky コンフィグ";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Localizable.strings]:
```suggestion
"Configuration Window Title" = "Unshakyの設定";
```

File: Unshaky/ja.lproj/MainMenu.strings [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+/* Class = "NSMenuItem"; title = "Check for Updates..."; ObjectID = "f5l-nJ-ecO"; */
+"f5l-nJ-ecO.title" = "アプデートをチェック中...";
+
+/* Class = "NSMenuItem"; title = "-"; ObjectID = "sJ6-3Q-q9Z"; */
+"sJ6-3Q-q9Z.title" = "-";
+
+/* Class = "NSMenuItem"; title = "Quit"; ObjectID = "ypt-Mu-uzT"; */
+"ypt-Mu-uzT.title" = "終了します";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/MainMenu.strings]:
```suggestion
"ypt-Mu-uzT.title" = "終了";
```

File: Unshaky/ja.lproj/Preference.strings [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+/* Class = "NSButtonCell"; title = "Set"; ObjectID = "juW-NR-znR"; */
+"juW-NR-znR.title" = "セット";
```
Comment (HackingGate) [inline Unshaky/ja.lproj/Preference.strings]:
```suggestion
"juW-NR-znR.title" = "適用";
```
セット(set) is a different meaning loanword, changed to 適用(apply). 
> [Sometimes a loanword develops a different meaning or usage than its donor word.](https://www.thelanguageisland.com/common-japanese-loanwords/)

**Comment by aahung** [comment]
Thanks for forgazer (translation), [HackingGate](https://github.com/HackingGate) (proofreading) to make this happen!

**Pull request merged**